### ISSUE TITLE
Program with non-trivial Flags Type does not compile

### ISSUE THREAD

**Issue by andreasewering** [marc136/snowpack-plugin-elm#3]
Hi,

I cannot seem to get a non-trivial program with Flags to compile. Here is a small example to reproduce:

```
module Main exposing (main)

import Browser
import Html exposing (..)


type alias Flags =
    { test : String
    }


main : Program Flags Model Msg
main =
    Browser.document
        { init = init
        , view = view
        , update = update
        , subscriptions = subscriptions
        }


type alias Model =
    { property : String
    }


init : Flags -> ( Model, Cmd Msg )
init _ =
    ( Model "", Cmd.none )


type Msg
    = Msg1
    | Msg2


update : Msg -> Model -> ( Model, Cmd Msg )
update _ model =
    ( model, Cmd.none )


subscriptions : Model -> Sub Msg
subscriptions _ =
    Sub.none


view : Model -> Browser.Document Msg
view _ =
    { title = "Document Title"
    , body =
        []
    }
```

When trying to start the dev mode or building, I get the following unhelpful error message:
```
[snowpack] ! verifying build...
[snowpack] ✘ C:\Users\aewering\elm\src\Main.elm
  Error: Parse error @:5203:3
    at parse (C:\Users\aewering\elm\node_modules\es-module-lexer\dist\lexer.cjs:1:402)
    at scanCodeImportsExports (C:\Users\aewering\elm\node_modules\snowpack\lib\rewrite-imports.js:12:29)
    at transformEsmImports (C:\Users\aewering\elm\node_modules\snowpack\lib\rewrite-imports.js:28:27)
    at Object.transformFileImports (C:\Users\aewering\elm\node_modules\snowpack\lib\rewrite-imports.js:75:16)
    at FileBuilder.resolveImports (C:\Users\aewering\elm\node_modules\snowpack\lib\commands\build.js:185:58)
    at C:\Users\aewering\elm\node_modules\snowpack\lib\commands\build.js:386:14
    at run (C:\Users\aewering\elm\node_modules\p-queue\dist\index.js:157:104)
    at PQueue._tryToStartAnother (C:\Users\aewering\elm\node_modules\p-queue\dist\index.js:105:17)
    at PQueue._next (C:\Users\aewering\elm\node_modules\p-queue\dist\index.js:48:14)
    at run (C:\Users\aewering\elm\node_modules\p-queue\dist\index.js:168:22)
```

The error message disappears as soon as I replace the Flags type by something that is not a record.
This only happens with the flags type, the model being a record does not cause any problems.

**Comment by andreasewering** [comment]
What does work as a workaround is using Json.Decode.Value as Flag Type and decoding it on init.
I don't mind that, but it should still work with a record, given that `elm make` succeeds

**Comment by marc136** [comment]
Hello @andreasewering, thank you for this bug report.

The issue should be fixed now with v0.4.1.

**Comment by andreasewering** [comment]
Thanks for the quick response and fix, works as intended 👍
